_posts/2015-03-16-CocoaPods.org-Take-Two.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "CocoaPods.org Take Two"
+author: orta
+categories: cocoapods.org releases website
+---
+
+TL;DR: [CocoaPods.org][1] version 2 has been released, with support for inline deep search results, and pod pages.
+
+<!-- more -->
+
+
+## This took way longer than expected.
```

Comment:
Starting the article with such title seems a bit strange and negative to me. How about "A bit of history" instead?